CHANGELOG.md

```diff
@@ -12,10 +12,12 @@ Bugfixes:
 
 Other improvements:
 
+- Run slowest tests last and print status updates (#72)
+
 ## [v6.0.1](https://github.com/purescript-contrib/purescript-string-parsers/releases/tag/v6.0.1) - 2021-05-11
 
 Other improvements:
-- Fix transitive dependencies errors found by Spago 0.20 (#71 by @milesfrain)
```

Comment:
Don't think it's necessary to name authors in the changelog.

Reply:
Oh, I didn't actually read the contrib template carefully enough in this case. Seems like there's a mismatch between core (no username) and contrib (with username).
- [Core](https://github.com/purescript/purescript-lists/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
- [Contrib](https://github.com/purescript-contrib/governance/blob/main/updater/templates/base/.github/PULL_REQUEST_TEMPLATE.md)

I think no-username is easier.

Reply:
We didn't used to associate any usernames with pull requests, but we'd like to going forward (for core, contrib, etc.). However, it _is_ optional if you'd rather not have your username added.